packages/cli-repl/test/e2e.spec.ts

```diff
@@ -939,9 +939,17 @@ describe('e2e', function() {
       expect(exitCode).to.equal(1);
     });
 
-    it('fails fast for ECONNREFUSED errors', async() => {
+    it('fails fast for ECONNREFUSED errors to a single host', async() => {
       const shell = TestShell.start({ args: [
-        '--port', '0'
+        '--port', '65535'
```

Comment:
Using 0 actually made this fail with a MongoParseError, without trying to connect at all